subjects/tuples_refs/README.md

```diff
@@ -25,11 +25,10 @@ Here is a program to test your functions
 use tuples_refs::*;
 
 fn main() {
-	let student = Student(20, "Pedro".to_string(), "Domingos".to_string());
-	println!("Student: {:?}", student);
-	println!("Student first name: {}", first_name(&student));
-	println!("Student last name: {}", last_name(&student));
-	println!("Student Id: {}", id(&student));
+    let student = Student(20, "Pedro".to_string(), "Domingos".to_string());
+    println!("Student's first name: {}", first_name(&student));
+    println!("Student's last name: {}", last_name(&student));
+    println!("Student's id: {}", id(&student));
```

Comment:
Change the output here as well according to the changes

Reply:
Once again I'm not sure I understood what needs to be changed, the exercise is still technically the same for both cases, we just switched owned with borrowed counterparts where the effect is exactly the same (because of coersion), but where the borrowed counterpart is more generic and thus suggested

Reply:
![image](https://github.com/user-attachments/assets/1715f898-20e0-4cea-a0fb-3b9c71721ebe)
this line of the output was removed on your changes

Reply:
Once more, this is deliberate. Removing `println!("Student: {:?}", student);` was intentional, as that line only executes if the struct `Student` derives `Debug`, which is not suggested or require anywhere in the exercise, so it seemed out-of-line to be suggested on the README.